src/components/WhereverWidget/WhereverWidget.tsx

```diff
@@ -0,0 +1,40 @@
+import { Colors } from "constants/constants";
+import { StyledWhereverWidget, StyledWhereverWidgetContainer } from "./styles";
+import {
+  NotificationFeed,
+  NotificationFeedProvider,
+  NotificationBell
+} from "@wherever/react-notification-feed";
+import { useEthers } from "@usedapp/core";
+import { WHEREVER_KEY } from "settings";
+
+const WhereverWidget = () => {
+  const { account, library } = useEthers();
+
+  if (!account) {
+    return null;
+  }
+
+  return (
+    <StyledWhereverWidgetContainer>
+      <NotificationFeedProvider
+        provider={library!}
+        theme={{
+          bellColor: Colors.turquoise,
+          backgroundColor: Colors.fieldBlue,
+          primaryColor: Colors.turquoise,
+          fontFamily: '"RobotoMono", sans-serif'
+        }}
```

Comment:
There is no way to set all those values through css? We are planning to stop using JS variables for colors.

Reply:
Currently no, but we are definitely planning on expanding the theming options for the widget based on our design partners' needs, so this will be open to discussion 💪